scaffolding-chef-infra/lib/windows/shared-chunk.rb

```diff
@@ -1,5 +1,5 @@
-cache_path '{{pkg.svc_data_path}}/cache'
-node_path '{{pkg.svc_data_path}}/nodes'
-role_path '{{pkg.svc_data_path}}/roles'
+cache_path 'build_time_pkg_svc_data_path/cache'
```

Comment:
Looking at CI I don't think this is rendering properly which is why the chef-client is failing.

Reply:
It looks like in the studio the `$pkg_svc_data_path` variable on windows is `\hab\svc\user-windows-default\data` but that is missing the drive letter. 😕